Redis.yml

```diff
@@ -0,0 +1,83 @@
+# Here is your stack preview
+# You can make advanced changes like modifying your VM,
+# installing packages, and running shell commands.
+
+provider:
+  aws:
+    access_key: '${var.aws_access_key}'
+    secret_key: '${var.aws_secret_key}'
+resource:
+  aws_instance:
+    # this is the name of your VM
+    redis:
+      # select your instance_type here: eg. c3.xlarge
+      instance_type: t2.medium
+      # select your ami (optional) eg. ami-xxxxx (it should be based on ubuntu 14.04)
+      ami: ''
+      # we will tag the instance here so you can identify it when you login to your AWS console
+      tags:
+        Name: '${var.koding_user_username}-${var.koding_group_slug}'
+      # on user_data section we will write bash and configure our VM
+      user_data: |-
+        # user_data Script:
+        set -e
+        export DEBIAN_FRONTEND=noninteractive
+        #Adding new source
+        #apt-key adv --keyserver pgp.mit.edu --recv-keys 5072E1F5
+        DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
+        CODENAME=$(lsb_release -cs)
+        #Getting user name and IP
+        export KODING_USER=${var.koding_user_username}
+        IPVAR=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
+        #Updating packages
+        apt-get update -y
+        #Installing Packages
+        apt-get install -y make gcc tcl
+        curl -O http://download.redis.io/releases/redis-3.2.3.tar.gz

```

Comment:
That will be easier but this PPA installs version 3.0.7, now the latest stable version is 3.2.6,  change it to be able to install different versions by using the variable $VERSION